Management/src/main/java/com/stack/jspdemo/model/Product.java

```diff
@@ -0,0 +1,64 @@
+package com.stack.jspdemo.model;
+
+import javax.validation.constraints.Size;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Why are you making a value class a spring component?

Reply:
It was unnecesary mark as a Component